jekyll/_cci2/getting-started.md

```diff
@@ -15,131 +15,190 @@ This document provides a step-by-step tutorial for getting your first successful
+   branch and start your first pipeline. **Note:** You can also manually add a
+   `config.yml` to your project root under a `.circleci` folder and push this to
+   your VCS provider. 
 
-## Setting up Your Build on CircleCI
+    ![Automatic Commit]( {{ site.baseurl }}/assets/img/docs/getting-started--auto-commit.png){:.img--bordered}
 
-1. For this step, you will need a CircleCI account. If you already have a CircleCI account then you can navigate to your [dashboard](https://circleci.com/dashboard), or if you are using  CircleCI Server substitute your hostname: `https://<your-circleci-hostname>.com/dashboard`. If you don't have an account yet, visit the CircleCI [signup page](https://circleci.com/signup) and click "Start with GitHub". You will need to give CircleCI access to your GitHub account to run your builds.
+1. You will be taken to the Pipelines page where you can view your newly running
+   build. 
 
-2. Next, you will be given the option of *following* any projects you have access to that are already building on CircleCI (this would typically apply to developers connected to a company or organization's GitHub account). On the next screen, you'll be able to add the repo you just created as a new project on CircleCI.
+## Digging Into Your First Pipeline
 
-3. To add your new repo, ensure that your GitHub account is selected in the dropdown in the upper-left, Select the Add Projects page, and find the repository you just created in the list (you can also search for it), then click the **Set Up Project** button next to it. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list.png)
+You should see your pipeline start to run automatically—and pass! So, what just
+happened? Click on the green `Success` button on your pipeline to investigate
+the following parts of the run: 
 
-4. On the following screen, choose a language from the dropdown to get a pre-populated config.yml file with suggested best-practices for your project. If the language you are using is not in the list, click the "Use Hello World" button at the bottom of the list. Copy the YAML configuration that appears and save it as a file called "config.yml" in a folder named `.circleci` in the root of your repository's directory. ![]( {{ site.baseurl }}/assets/img/docs/CircleCI-add-new-project-list-2.png)
+![First Successful Pipeline]( {{ site.baseurl }}/assets/img/docs/getting-started--first-success.png){:.img--bordered}
 
-5. Git push your new  `config.yml` file to your VCS. You are ready to start building! Click the button "Start Building" and confirm in the modal that your repository is ready to build.
+1. **What workflows ran?**: After clicking "Success", we are taken to a page
+   listing the jobs that ran. If this is your first build, you probably only ran
+   **one job**  (which automatically runs inside **one workflow**).  In our
+   case, we only ran one job, called `Build`. Click on `Build` and let's
```

Comment:
In the image the job is `welcome/run`, rather than `build`

Reply:
fixed!